app/serializers/post_serializer.rb

```diff
@@ -256,6 +256,7 @@ def trust_level
   def reply_to_user
     {
       username: object.reply_to_user.username,
+      name: object.reply_to_user.name,
```

Comment:
It would be great if we could add a serializer spec for `reply_to_user` (`git blame` tells me this method is from Discourse's initial release!).